.gitignore

```diff
@@ -58,3 +58,5 @@ scripts/curl.fish
 curl_fuzzer
 curl_fuzzer_seed_corpus.zip
 libstandaloneengine.a
+cmake-build-debug
+.idea
```

Comment:
I don't think we should ignore IDE files for the following reasons:
1) There are just too many editors
2) The committer is responsible for adding these files. Also there are ways to globally ignore some files with git locally